recipes/parallelproj/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set name = "parallelproj" %}
+{% set version = "1.2.5" %}
+{% set build_number = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gschramm/parallelproj/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 411fe7552106db2e20487f2429907bfac641020469b0e7593134975345148cf5
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe
+  # is exactly the same for all platforms. It is okay if the dependencies are
+  # not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python
+  # for more details.
+  # noarch: python
+
+  number: {{ build_number }}
+  skip: True  # [win]
+  skip: True  # [osx]
+
+  # If the installation is complex, or different between Unix and Windows,
+  # use separate bld.bat and build.sh files instead of this key. By default,
+  # the package will be built for the Python versions supported by conda-forge
+  # and for all major OSs. Add the line "skip: True  # [py<35]" (for example)
+  # to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit
+  # to Windows.
+
+outputs:
+  - name: parallelproj
+    build:
+      skip: True  # [cuda_compiler_version != "None"]
+    script: build_openmp.sh
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake >=3.23
+        - ninja
+        - libgomp  # [linux]
+        - llvm-openmp  # [osx]
+      host:
+      run:
+        - libgomp  # [linux]
+        - llvm-openmp  # [osx]
+    test:
+      requires:
+        - cmake >=3.23
+      commands:
+        - ctest -VV
+
+  - name: parallelproj-gpu
+    build:
+      skip: True  # [cuda_compiler_version == "None"]
+      missing_dso_whitelist:
+        - $RPATH/libgomp.so.1
+        - $RPATH/libstdc++.so.6
```

Comment:
It seems to me like you should add libgomp / llvm-openmp to `host:`, then you shouldn't need this.

Reply:
@h-vetinari @jaimergp I added to libgomp / llvm-openmp to ```host:``` requirements, but the build with the Error: ```ERROR (parallelproj-gpu,lib/libparallelproj_cuda.so.1.2.5): $RPATH/libgomp.so.1 not found in packages, sysroot(s) nor the missing_dso_whitelist```. Any idea what I am doing wrong?